doc/user-manual/language/cubical.lagda.rst

```diff
@@ -413,7 +413,7 @@ They satisfy the following equalities:
 
   outS (inS a) = a
 
-  inS {u = u} (outS {u = u} a) = a
+  inS {φ = φ} (outS {φ = φ} a) = a
```

Comment:
Could this be transformed into type-checked Agda code so that Agda checks whether what is written there is correct?
```agda
inS-outS : forall {...} -> inS {φ = φ} (outS {φ = φ} a) == a
inS-outS = refl
```